2020F-usfca-btec-640.github.io/index.markdown

```diff
@@ -0,0 +1,6 @@
+---
+# Feel free to add content and custom Front Matter to this file.
```

Comment:
This file should not be a part of the PR. Please remove it so we don't have to deal with merge conflicts.